sql/utils.sql

```diff
@@ -13,6 +13,18 @@ select case when type like any(array['character', 'character varying', 'text'])
             else 'string' end;
 $$;
 
+create or replace function postgrest_unfold_comment(comm text) returns text[]
```

Comment:
Is there a unit test for this function?

Reply:
Ah, no. In fact, I think none of the functions have unit tests. 

But it makes sense for the functions inside  `utils.sql` to have them, at least.  I think they could be added in another PR. I'll open an issue for those.